### ISSUE TITLE
Instructions to launch exercise with GPU is missing

### ISSUE THREAD

**Issue by iamarkaj** [JdeRobot/RoboticsAcademy#1986]
Broken link: https://jderobot.github.io/gpu_acceleration

![img1](https://user-images.githubusercontent.com/40225765/227767164-a6d74a9b-af1f-4fb7-9a9e-b8b1ddac7fd4.png)

![img2](https://user-images.githubusercontent.com/40225765/227767167-add4486c-e0fc-4d64-b391-9c77b493ec29.png)

**Comment by pawanw17** [comment]
Hi thanks for reporting, the correct link should have been https://jderobot.github.io/RoboticsAcademy/gpu_acceleration/

**Comment by jmplaza** [comment]
Uhm, instead of having a specific webpage for this, perhaps we should add the GPU launching instructions to the ["Installation and running instructions" webpage](https://jderobot.github.io/RoboticsAcademy/installation/). Just to have all the information regarding using RoboticsAcademy in a single page....

Maybe we could rename that webpage as "User guide", don't you think?

**Comment by pawanw17** [comment]
It is there as well in the end. as a link ... more on this here: https://github.com/JdeRobot/RoboticsAcademy/pull/1972#pullrequestreview-1345535208

**Comment by pawanw17** [comment]
A quick fix to this issue would be to change /gpu_acceleration to /RoboticsAcademy/gpu_acceleration in the pages where it is used eg: https://raw.githubusercontent.com/JdeRobot/RoboticsAcademy/gh-pages/_pages/installation.md here however that'd make it not run in the development env.

**Comment by pawanw17** [comment]
@pariaspe @jmplaza  Do you think I should remove these lines from the exercises?

```
Enable GPU Acceleration
Follow the advanced launching instructions from [here](gpu_acceleration).
```
or replace the link with /user_guide... If I do replace it then there'll be the same problem facing currently


https://github.com/JdeRobot/RoboticsAcademy/blob/gh-pages/_pages/exercises/AutonomousCars/autoparking.md#enable-gpu-acceleration

**Comment by jmplaza** [comment]
Yes, in the exercise webpages a link to /user_guide page should be included, which has to be a permalink (replacing /installation).

In the user_guide the instructions with the URL to use from the browser to start using RoboticsAcademy should be clarified too.

I guess we don't need a different environment for users and for developers of RoboticsAcademy. The developer may modify the source code of the webserver or a given exercise, and launch the webserver at the same port. Anyway we could include a /developer_guide webpage with the instructions to deploy RoboticsAcademy without creating a RADI, just launching django locally....

**Comment by pawanw17** [comment]
> Yes, in the exercise webpages a link to /user_guide page should be included, which has to be a permalink (replacing /installation).
> 
> In the user_guide the instructions with the URL to use from the browser to start using RoboticsAcademy should be clarified too.
> 
> I guess we don't need a different environment for users and for developers of RoboticsAcademy. The developer may modify the source code of the webserver or a given exercise, and launch the webserver at the same port. Anyway we could include a /developer_guide webpage with the instructions to deploy RoboticsAcademy without creating a RADI, just copying the repositories and launching django locally....

Sorry I didn't mention clearly, if I add link /user_guide in the exercises page, the Website will work in development environment 127.0.0.1:4000/user_guide

however in the production it'll redirect to https://jderobot.github.io/user_guide, which will give page does not exist error,

If I instead add the link /RoboticsAcademy/user_guide in the exercises pages, then it'll work in the production environment but not in the development environment since 127.0.0.1:4000/RoboticsAcademy/ does not exist.... This is the issue currently facing with the gpu_acceleration page

**Comment by pawanw17** [comment]
Um to edit exercises documentation I go to gh-pages branch and run
`bundle exec jekyll serve`

Upon doing so on 127.0.0.1:4000 this is the website django server not the RADI one

![image](https://user-images.githubusercontent.com/40732709/228205336-56a0f203-8307-4678-af14-bc0e5f0c1b01.png)
![image](https://user-images.githubusercontent.com/40732709/228205622-41f30da9-530b-435d-b3fe-a7b9ff57ead0.png)

To edit exercise documentation I have used this http://jderobot.github.io/RoboticsAcademy/contribute/#jekyll-local-installation

I am not running the RADI, or the RADI django server which runs on 8000

**Comment by jmplaza** [comment]
@pawanw17 check my previous updated comment. 

There was a confusion in our conversation, please notice there are two sides here: 
(a) DOCUMENTATION 
the update of GitHub Pages documentation  (calling the local deployment of GitHub Pages with Jekyll templates a "development environment" is too ambitious...) 

(b) OPERATING ROBOTICSACADEMY 
the development environment for RoboticsAcademy developers (refining of exercises, of the django webserver, creation of new exercises).

Hope it is more clear now.

**Comment by pawanw17** [comment]
yes understood, the cause of confusion was "development environment" XD, I'll make a PR soon, Thanks for clearing things out!

**Comment by jmplaza** [comment]
https://forum.unibotics.org/ is up and running AFAIK

**Comment by pawanw17** [comment]
yeah but the website redirects to https://forum.jderobot.org/c/english/roboticsacademy

**Comment by jmplaza** [comment]
please just correct the link to https://forum.unibotics.org/ in the PR

**Comment by jmplaza** [comment]
Fixed in #1988 and #1989. Well done @pawanw17 !
